client-app/ui-kit/components/molecules/address-form/vc-address-form.vue

```diff
@@ -222,47 +213,51 @@ const slotsData = computed(() => ({
 }));
 
 const emailRules = computed(() => {
-  let rules = yupString().max(64).email().nullable();
+  let rules = yup.string().trim().max(64).email().nullable();
   if (props.withPersonalInfo && props.requiredEmail) {
     rules = rules.required();
   }
   return toTypedSchema(rules);
 });
 
 const phoneRules = computed(() => {
-  let rules = yupString().max(64).nullable();
+  let rules = yup.string().trim().max(64).nullable();
```

Comment:
put it numbers in constant on the top or settings please

Reply:
Same as above